packages/comet/src/error.ts

```diff
@@ -73,6 +73,16 @@ export class CometErrorHandler {
             status: 400,
             headers: { 'content-type': 'application/json' }
           })
+        case ErrorType.InvalidCompatibilityDate:
+          return new Response(JSON.stringify({ success: false, error: 'Invalid compatibility date' }), {
```

Comment:
I'm not sure about including `{ success: false }`, a client can probably ascertain the success of their request by checking if the status code is 200-299, or just `Response.ok`